src/app/services/preprocessing/steps/feature_selection.py

```diff
@@ -1,11 +1,16 @@
 import polars as pl
 
 from app.services.preprocessing.config.feature_selection_config import SelectionConfig
+from app.settings import Settings
 
 
-def select_features(
-    df: pl.DataFrame, config: SelectionConfig
-) -> tuple[pl.DataFrame, pl.Series]:
-    X = df.drop(config.features_to_drop)
-    y = df[config.target_feature]
-    return X, y
+def select_features(df: pl.DataFrame, config: SelectionConfig) -> dict[str, str | int]:
+    df = df.drop(config.features_to_drop)
+    save_path = Settings.UPLOAD_DIRECTORY / "train.csv"
+    df.write_csv(save_path)
+
+    return {
+        "message": "Feature selection completed",
+        "total_features": df.shape[1],
+        "total_rows": df.shape[0],
+    }
```

Comment:
maybe if you are going to use this type on return in several parts, you should create a output schema.

```suggestion
    class PreprocessingResult(BaseModel):
        message: str
        total_features: int
        total_rows: int
```
And then return an object PreprocessingResult